packages/node_modules/@node-red/editor-client/src/js/ui/common/typedInput.js

```diff
@@ -46,6 +46,12 @@
             opacity: 0.3
         }).appendTo(container);
         this.elementDiv.show();
+        if (!this.input.hasClass('red-ui-autoComplete')) {
+            this.input.autoComplete({
+                search: contextAutoComplete({ input: that }),
+                minLength: 0
+            })
+        }
```

Comment:
`contextLabel` is not called if only one storage is used - autocomplete is disabled

Reply:
This is now fixed. I've improved how the context autocomplete is initialised - doesn't need to be done as part of the `contextLabel` anymore.